packages/driver/src/cypress/mocha.ts

```diff
@@ -35,13 +35,22 @@ const suiteAfterEach = Suite.prototype.afterEach
 delete (window as any).mocha
 delete (window as any).Mocha
 
-function invokeFnWithOriginalTitle (ctx, originalTitle, mochaArgs, fn, _testConfig) {
-  const ret = fn.apply(ctx, mochaArgs)
+type MochaArgs = [string, Function | undefined]
+function createRunnable (ctx, fnType: 'Test' | 'Suite', mochaArgs: MochaArgs, runnableFn: Function, testCallback: Function | string, _testConfig?: Record<string, any>) {
+  const runnable = runnableFn.apply(ctx, mochaArgs)
 
-  ret._testConfig = _testConfig
-  ret.originalTitle = originalTitle
+  // attached testConfigOverrides will execute before `runner:test:before:run` event
+  if (_testConfig) {
+    runnable._testConfig = _testConfig
+  }
+
+  if (fnType === 'Test') {
+    // persist the original callback so we can send it to the cloud
+    // to prevent it from being registered as a modified test
+    runnable.body = testCallback.toString()
```

Comment:
Since testCallback is always defined - either a function or a string (usually an empty string) - it looks like we could remove the if, and just always `runnable.body = testCallback.toString()`, regardless of fnType. Then we'd always have runnable.body defined, and the shape would be more consistent.

Or even better, set `''` as a default argument, and skip the `|| ''`s below in overloadMochaFnForConfig.

Reply:
We only want to set on Test body for now.